### PR TITLE
Add contact form -pair programming

### PR DESCRIPTION
**Pull request by henrycadenaec**
Add index.html
Add style.css
Add background image to the contact section
Add footer efect

Please consider: The webhint error (not successful checked) will not be taken into account by the reviewers. Thank you very much.

### CODE REVIEW

**Review by shahadat3669 — CHANGES_REQUESTED** [review]
Hi @henrycadenaec, @Fentahunn , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.



## Invalid Code Review Request

You have submitted a project, that is invalid because you did not follow the GitHub flows. It's important to follow the GitHub flow properly. I recommend using "main" as the base branch name instead of "master". This follows industry best practices and promotes inclusive and respectful language. You can refer to the [GitHub Flow guide](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md) for more information.

By doing so, you will align with established practices and ensure the smooth progression of your project. Check out this [official document](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-branches-in-your-repository/renaming-a-branch) for further help. :+1: and Keep up the good work! 👍 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -308,5 +308,58 @@ <h4>LET'S CONNECT</h4>
         </div>
       </aside>
     </section>
+
+    <section class="contact">
+      <div class="background-contact">
+        <article class="main-background-image-contact">
+          <aside class="main-background-setup-contact">
+            <h2 class="h2-white">Contact me</h2>
```
Comment (shahadat3669) [inline index.html]: 👏 Great work on your project! Upon reviewing it, I noticed that your contact and other sections are functioning correctly, which is commendable. However, there is a minor issue with the font size of your section titles. They do not match the design specifications. 

To ensure consistency and adherence to the design, I recommend updating the font size of your section titles. Compare them closely with the design and make the necessary adjustments to match the desired font size. 😄 

Keep up the good work and remember to pay attention to even the smallest details. By aligning the font size with the design, you will create a visually cohesive and polished user experience. 💪🎨

**Review by shahadat3669 — CHANGES_REQUESTED** [review]
Hi everyone, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
✅ Followed GitHub flow.
✅ Descriptive PR title & summary.
✅ Professional README file.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
-  [Optional] In the "Table of contents" of your Readme file you have a heading "FAQ" but I notice that there is no "FAQ" section available in your Readme file. For this reason, please consider removing it from the Table of Contents, ensuring consistency between the content and the navigational elements.
image

Cheers and Happy coding!👏👏👏

## Have any doubt ❓

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

## ⚠️ WARNING ⚠️
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by henrycadenaec** [comment]
Thank you very muach @shahadat3669

**Review by usorfaitheloho — APPROVED** [review]
Hi @henrycadenaec  , 
# STATUS: APPROVED! 🥇 💯 
- [x] Very Nice Design, Keep it up!!
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by henrycadenaec** [comment]
Thank you very much !!!

**Pull request merged**